modules/platform/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/main/java/org/nuxeo/elasticsearch/audit/ESAuditBackend.java

```diff
@@ -715,8 +715,11 @@ protected void ensureUIDSequencer(ESClient esClient) {
         request.source(new SearchSourceBuilder().query(QueryBuilders.matchAllQuery())
                                                 .aggregation(AggregationBuilders.max("maxAgg").field("id")));
         SearchResponse searchResponse = esClient.search(request);
-        Max agg = searchResponse.getAggregations().get("maxAgg");
-        long maxLogEntryId = (long) agg.getValue();
+        Aggregation agg = searchResponse.getAggregations().get("maxAgg");
+        long maxLogEntryId = 0;
+        if (agg.getMetadata() != null && agg.getMetadata().containsKey(Aggregation.CommonFields.VALUE)) {
+            maxLogEntryId = (long) agg.getMetadata().get(Aggregation.CommonFields.VALUE);
+        }
```

Comment:
Could we put `agg.getMetaData()` in a local `var` at least to avoid repetition? It could be done directly with

    var aggMetadata = searchResponse.getAggregations().get("maxAgg").getMetaData();

But I would also really want to advocate `getOrDefault`, it's a habit that's good to have.

Reply:
ok will do that another time, the CI cannot validate a new change before merging